### ISSUE TITLE
No Windows Binaries

### ISSUE THREAD

**Issue by coreybutler** [nodejs/corepack#53]
I run [nvm-windows](https://github.com/coreybutler/nvm-windows) and have had a few users trying to use Node 16.9.0 with corepack. It's not currently clear how this will be shipping for Windows, and there don't appear to be any builds for Windows at https://github.com/nodejs/corepack/actions/runs/276463953. As it currently stands, it appears the only way to use corepack on Windows is through an npm global installation.

_In case it helps, here is some background info about how version management is handled on Windows:_

Currently, Windows version managers source node and their associated npm versions from https://nodejs.org/dist/index.json. The paths are derived using the zip archives from each of their respective repos (i.e. https://nodejs.org/dist/) and https://github.com/npm/cli/archive/ respectively. There is no special installation process for any version of Node. In NVM4W, a symlink is configured to point to the appropriate version, then it remains hands-off.

If corepack is going to work with version managers, it cannot be reliant on the Node installer. The Node installer is not used in version managers on Windows (this is true for Nodist as well) because it uninstalls prior versions and fails if a newer version already exists. In other words, it only allows one version to be installed at a time. See https://github.com/nodejs/node/issues/4603 for details. For npm, NVM4W currently copies the npm/npx commands to the node root, making them accessible as global commands. If corepack shipped a bootstrap command like this, we could do the same thing.

Bottom line: I'm trying to figure out how to bootstrap corepack so Windows recognizes it for the active version of Node.

**Comment by coreybutler** [comment]
Please ignore... I am just now seeing that corepack ships the cmd file as I mentioned above.

**Comment by merceyz** [comment]
See https://github.com/coreybutler/nvm-windows/issues/660#issuecomment-917073598 and https://github.com/nodejs/corepack/issues/52#issuecomment-917057797
